gpu-operator/release-notes.rst

```diff
@@ -31,8 +31,134 @@ See the :ref:`GPU Operator Component Matrix` for a list of components included i
 
    GPU Operator beta releases are documented on `GitHub <https://github.com/NVIDIA/gpu-operator/releases>`_. NVIDIA AI Enterprise builds are not posted on GitHub.
 
+
 ----
 
+.. _v24.3.0:
+
+24.3.0
+======
+
+.. _v24.3.0-new-features:
+
+New Features
+------------
+
+
+* Added support to enable NVIDIA GDRCopy v2.4.1.
+
+  When you enable support for GDRCopy, the Operator configures the GPUCopy Driver container image
+  as a sidecar container in the GPU driver pod.
+  The sidecar container compiles and installs the gdrdrv Linux kernel module.
+  This feature is supported on Ubuntu 22.04 and RHCOS operating systems and on X86_64 and ARM64 architectures.
+
+  Refer to :ref:`Chart Customization Options` for more information about the ``driver.gdrcopy`` field.
+
+* Added support for the NVIDIA Data Center GPU Driver version 550.54.15.
+  Refer to the :ref:`GPU Operator Component Matrix`
+  on the platform support page.
+
+* Added support for the following software component versions:
+
+    - NVIDIA Container Toolkit version v1.15.0
+    - NVIDIA MIG Manager version v0.6.0
+    - NVIDIA Driver Manager for K8s v0.6.8
+    - NVIDIA Kubernetes Device Plugin v0.15.0
+    - DCGM 3.3.3-1
+    - DCGM Exporter 3.3.5-3.4.0
+    - Node Feature Discovery v0.15.4
+    - NVIDIA GPU Feature Discovery for Kubernetes v0.15.0
+    - NVIDIA MIG Manager for Kubernetes v0.6.0
+    - NVIDIA KubeVirt GPU Device Plugin v1.2.6
+    - NVIDIA vGPU Device Manager v0.2.5
+
+* Added support for Kubernetes v1.29 and v1.30.
+  Refer to :ref:`Supported Operating Systems and Kubernetes Platforms`.
+
+* Added support for NVIDIA GH200 Grace Hopper Superchip as a generally available feature.
```

Comment:
I don't think we support this yet. We need to update mig manager to support this device and update gpu-operator to indicate that GH200 is mig-capable

Reply:
@mikemckiernan unresolving this as we need to remove this from the release notes

GH200 has not been verified on gpu-operator

Reply:
Sorry nvm. I see that this has been fixed

Reply:
I checked with Francis on this one -- we do indeed want to call out support for the GH200 super chip, as R550 adds official support with it, and our QA covered it. Apologies for any confusion here.

@mikemckiernan let's add this bullet back. Will need to revert this revert :) https://github.com/NVIDIA/cloud-native-docs/pull/44/commits/c0fff0e61be38146eed1d58b5739d75baf2a8ad8